src/main/resources/headers.config

```diff
@@ -0,0 +1,70 @@
+allocateTarget                      = markupDeadline
```

Comment:
If these are just used for human-readable CSV headers, there's no reason for them to be camel case: they can be real words with spaces instead.